projects/schematics/src/shared/constants.ts

```diff
@@ -848,6 +848,8 @@ export const DEFAULT_USER_GROUP_ROUTING_CONFIG =
   'defaultUserGroupRoutingConfig';
 export const DEFAULT_USER_ROUTING_CONFIG = 'defaultUserRoutingConfig';
 
+export const CUSTOMER_TICKETING_SCSS_FILE_NAME = 'customer-ticketing.scss';
```

Comment:
need to delete as it's a duplicate with the `customer-ticketing-schematics-config.ts`